source/_integrations/sensor.markdown

```diff
@@ -22,6 +22,7 @@ The type of data a sensor returns impacts how it is displayed in the frontend. T
 - **carbon_dioxide**: Carbon Dioxide in CO2 (Smoke)
 - **carbon_monoxide**: Carbon Monoxide in CO (Gas CNG/LPG)
 - **current**: Current in A.
+- **date**: Date object or date string (ISO 8601).
```

Comment:
The state returned from the state machine will always be an isoformatted date string. The incoming data that we convert in the sensor platform could be a date object.

I'm not sure what case this list is describing.

Reply:
The state, so it is a date string.

```suggestion
- **date**: Date string (ISO 8601).
```